README.md

```diff
@@ -70,9 +70,9 @@ LuaJIT [3].
 
 This pass implements a classical algorithm originally designed by M. N. Wegman
 and F. K. Zadeck [4] for SSA form. Unification of data and control dependencies
-made its implementation even simple. Despite of constant propagation itself
+made its implementation even simpler. Despite of constant propagation itself
 this pass also performs global Copy Propagation and re-applies the folding rules.
-At the end all the “dead” instructions (instructions that result are not used)
+At the end all the “dead” instructions (instructions that are not used)
```

Comment:
We can't check if the instruction is used or not. We may check if the value produced by the instruction (its result) is used. may be: **which results**?

Reply:
> Sorry for this one too. I reworded to keep the original meaning.

No problems and thank you.